src/screens/benefit/Details.tsx

```
@@ -261,6 +261,17 @@ const BenefitsDetails: React.FC = () => {
 					}
 				: (authUser ?? undefined);
 
+			// Calculate age from dob if present
+			const formData =
+				baseFormData && (baseFormData as any)?.dob
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)